### DISCUSSION TITLE
PyVista not working: black screen but no error message

### DISCUSSION THREAD

**Discussion by athcal** [pyvista/pyvista#6409] [Q&A]
### First check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the PyVista documentation, with the integrated search.
- [X] I already searched in Google "How to X in PyVista" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I will NOT make duplicate discussions or issues in the future.

### Commit to Help

- [X] I commit to help with one of those options.

### Sample Code What is the problem, question, or error?
Write a short description telling me what you are doing, what you expect to happen, and what is currently happening.

```python
import pyvista as pv
from pyvista import examples
vol = examples.download_knee_full()
pl = pv.Plotter(off_screen = False)
pl.add_volume(vol, cmap="bone", opacity="sigmoid", show_scalar_bar=False)
pl.enable_depth_peeling()

pl.show()

#figure = pl.screenshot("initial1.png")
```

### Description

Hello everyone,

I've a problem with pyvista. When I run this simple code,

```
import pyvista as pv
from pyvista import examples
vol = examples.download_knee_full()
pl = pv.Plotter(off_screen = False)
pl.add_volume(vol, cmap="bone", opacity="sigmoid", show_scalar_bar=False)
pl.enable_depth_peeling()

pl.show()
#figure = pl.screenshot("initial1.png")
```

 pyvista opens but the screen is all black. However, no error message is produced.

![Screenshot 2024-07-23 183128](https://github.com/user-attachments/assets/824cf5d7-3f31-4221-b9ee-8b7e67912985)

 If I try to save a png screenshot, just a black image is saved. Does someone know how to fix this?

![initial1](https://github.com/user-attachments/assets/042e272b-11ca-48f0-93fb-88cf59646859)

### System Information

```shell
Date: Wed Jul 24 17:01:58 2024 CEST

                OS : Linux (Ubuntu 22.04)
            CPU(s) : 4
           Machine : x86_64
      Architecture : 64bit
       Environment : Python
        GPU Vendor : Microsoft Corporation
      GPU Renderer : D3D12 (Intel(R) UHD Graphics)
       GPU Version : 4.1 (Core Profile) Mesa 23.2.1-1ubuntu3.1~22.04.2
  MathText Support : True

  Python 3.12.4 | packaged by conda-forge | (main, Jun 17 2024, 10:23:07) [GCC
  12.3.0]

           pyvista : 0.44.1
               vtk : 9.3.1
             numpy : 1.26.4
        matplotlib : 3.8.4
            scooby : 0.10.0
             pooch : 1.8.2
            pillow : 10.4.0
```

**Comment by user27182** [comment]
Do you still have this issue if you comment out  `pl.enable_depth_peeling()` ? I'm asking because there are a few issues with depth peeling https://github.com/search?q=repo%3Apyvista%2Fpyvista+depth+peeling&type=issues

**Reply by athcal** [reply]
Yes, I have the same problem also in that case
